addons/scratchstats/userscript.js

```diff
@@ -114,7 +114,14 @@ export default async function ({ addon, msg, console }) {
                   },
                 },
                 y: {
-                  stepSize,
+                  precision: 0,
+                  ticks: {
+                    callback: function (stepSize) {
```

Comment:
The `stepSize` variable defined above (not the function argument) is now unused. Can you remove line 91?

Reply:
Done